specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json

```diff
@@ -7171,6 +7213,43 @@
         }
       }
     },
+    "RegistrationInfoList": {
+      "type": "object",
+      "title": "RegistrationInfoList",
+      "description": "List of RegistrationInfo definitions.",
+      "properties": {
```

Comment:
>properties [](http://example.com/codeflow?start=7&length=10)

why is this an array ? Its a bit confusing since this is listing the tokens , but each individual token only has "expirationtime" and "token" so what is the difference between elements of the array ? do you need another property to identity the array elements ?

Reply:
This provides the same info available in retrieveRegistrationToken, we just needed a POST endpoint that was digestible by arm/bicep templates for a security fix. The active registration token will be identifiable based on the expiration time.